### ISSUE TITLE
Drinking potions makes me thirsty

### ISSUE THREAD

**Issue by Rufflikerex** [Glitchfiend/ToughAsNails#567]
Drinking any Minecraft potion will give me the thirst effect.
Please fix this.

**Comment by paulgb434** [comment]
Issue still isn't fixed in 3.1.0 - 129 unfortunately

**Comment by CorneliaXaos** [comment]
~~A hacky fix you could do is to go to the file: `<modpack_root>/config/toughasnails/drink_stats.json` and add the following to the JSON array:~~

```json
  {
    "item": {
      "name": "minecraft:potion",
      "metadata": 0
    },
    "thirst": 4,
    "hydration": 0.3,
    "poisonChance": 0.0
  }
```

~~You might need a comma in there....~~

...

Actually, that might not work..  I think ToughAsNails will intercept the potion usage before that data can be applied...

Also.. after looking at the code a bit more it doesn't look like that can happen at all unless something with Forge changed.  `minecraft:potion` items that have **NO** effects should give the player thirst, but potions that have effects shouldn't.
